docs/business-plan/tools/build_final_business_plan_docx.py

```
@@ -0,0 +1,1292 @@
+    add_note_box(
+        document,
+        "서비스 정의",
+        "질문을 알아듣고, 서류를 읽고, 행정기한을 놓치지 않도록 돕는 충북형 AI 비자 동행 서비스다.",
+    )
+
+    # 6쪽 — 데이터와 판정
+    add_page_break(document)
+    document.add_heading("3-3. AI 행정·안전 라우팅과 데이터 신뢰 구조", level=2)
+    add_body(
+        document,
+        "핵심 원칙은 ‘AI는 무엇을 물었는지 파악하고, 무엇을 답할지는 검수된 데이터가 결정한다’는 역할 분리다. 구축 대상 공식 원천문서 24건 전체를 파싱해 원문·근거·규칙·서비스의 4계층으로 구조화했으며, 모든 질문은 위험 신호·사용자 유형·지역·비자·언어로 분류된 뒤 제한형 조회와 결정론적 코드로 처리된다.",
```

Ruff (0.16.2)

[warning] String contains ambiguous `‘` (LEFT SINGLE QUOTATION MARK). Did you mean ``` (GRAVE ACCENT)?

(RUF001)